openlibrary/components/AuthorIdentifiers.vue

```diff
@@ -29,12 +29,24 @@
 </template>
 
 <script>
+const uuidV4Validator = /^[\da-f]{8}-[\da-f]{4}-[\da-f]{4}-[\da-f]{4}-[\da-f]{12}$/
+const genericNumericValidator = /^[0-9]+$/
 const identifierPatterns  = {
```

Comment:
This is the constant used for autoselection, right? How does this work when some patterns match multiple identifiers?

Also, this is maybe scope/feature creep for the PR, since this seems outside the scope of the issues referenced? Might be better to take these bits out to a separate PR. (See also [‘Keep Your Commits “Atomic”’](https://www.freshconsulting.com/insights/blog/atomic-commits/) which can be applied to PRs too.)

Reply:
Fair point! I included it here because I was using this feature to test the validity of the identifiers being imported.